cms/static/sass/elements/_forms.scss

```diff
@@ -172,8 +172,97 @@ form {
   }
 }
 
+// +Forms - License selector UI
+// ====================
+.license-img {
+  padding: 4px;
+}
+
+ul.license-types {
+  text-align: middle;
+  vertical-align: middle;
+  display: inline-block;
+
+  li {
+    display: inline-block;
+  }
+
+  .action.license-button {
+    @include grey-button;
+    @extend %t-action2;
+
+    display: inline-block;
+
+    text-align: center;
+
+    width: 220px;
+    height: 40px;
+
+    cursor: pointer;
+
+    &.is-selected {
+      @include blue-button;
+    }
+  }
+
+  .tip {
+    @extend %t-copy-sub2;
+  }
+}
+.wrapper-license-options {
+  margin-bottom: 10px;
+
+  .tip {
+    @extend %t-copy-sub2;
+  }
+  ul.license-options {
+    padding-bottom: 10px;
+
+    li {
+      line-height: 1.5;
+      border-bottom: 1px solid #EEE;
+      padding: ($baseline / 2) 0 ($baseline * 0.4);
+
+      &.is-clickable {
+        cursor: pointer;
+      }
+      &.last {
+        border-bottom: none;
+      }
+
+      input[type=checkbox] {
+        vertical-align: top;
+        width: auto;
+        min-width: auto;
+        height: auto;
+        border: 0;
```

Comment:
you can remove the four rules above, they're being overwritten by other rules anyways - width, min-width, height, border.